supervisor/backups/backup.py

```diff
@@ -67,6 +67,12 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 
+@lru_cache
+def _backup_file_size(backup: Path) -> int:
+    """Get backup file size."""
+    return backup.stat().st_size if backup.is_file() else 0
```

Comment:
We should get the size when scanning the backup and cache the information. But it used to work like this before, so let's do one thing at a time.